sdk/formrecognizer/azure-ai-formrecognizer/CHANGELOG.md

```diff
@@ -0,0 +1,10 @@
+# Change Log azure-ai-formrecognizer
+
+## 1.0.0b1 (Unreleased)
+
+Version (1.0.0b1) is the first preview of our efforts to create a user-friendly and Pythonic client library for Azure Form Recognizer. For more information about this, and preview releases of other Azure SDK libraries, please visit
```

Comment:
Should we mention it's a replacement of https://pypi.org/project/azure-cognitiveservices-formrecognizer/ ?

Reply:
Added this, good suggestion